pkg/manager/client.go

```diff
@@ -6,11 +6,18 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"runtime"
 
+	"github.com/acorn-io/runtime/pkg/version"
 	"github.com/sirupsen/logrus"
 )
 
-var ErrTokenNotFound = fmt.Errorf("token not found")
+var (
+	ErrTokenNotFound = fmt.Errorf("token not found")
+	ErrForbidden     = fmt.Errorf("forbidden")
+
+	userAgent = fmt.Sprintf("acorn/%s (%s; %s)", version.Get().String(), runtime.GOOS, runtime.GOARCH)
```

Comment:
Nit: s/acorn/runtime/g 😄

Reply:
The client in this case is the `acorn` CLI. I don't think a user would know anything about `runtime` if they are only using the CLI with manager.

Reply:
Good point